docs/swarm/swarm-tutorial/create-swarm.md

```diff
@@ -0,0 +1,76 @@
+<!--[metadata]>
++++
+title = "Create a Swarm"
+description = "Initialize the Swarm"
+keywords = ["tutorial, cluster management, swarm"]
+[menu.main]
+identifier="initialize-swarm"
+parent="swarm-tutorial"
+weight=12
++++
+<![end-metadata]-->
+
+## Create a Swarm
+
+After you install Docker on your networked machines and start the Docker Engine
+daemon, you're ready to create a Swarm. If you haven't already, read the through
+the [tutorial setup](tutorial-setup.md).
+
+1. Open a terminal and ssh into the machine where you want to run your manager
+node. For example, the tutorial uses a machine named `manager1`.
+
+2. To create a new Swarm, run the following command:
```

Comment:
I'd make this imperative so you can combine the follow-on into one line:
"Create a swarm and initialize a new node by running `command`."